console/frontend/src/main/frontend/src/app/views/status/status.component.scss

```diff
@@ -0,0 +1,13 @@
+tr.hasErrorsSummary {
+  color: #ed5565;
```

Comment:
Zou het niet handig zijn om vaker gebruikte kleur codes in variables bestand te declareren?

Reply:
Dat zou inderdaad handig zijn. Maar dat is iets wat we bij een refactor gaan doen, dan kunnen we ook alle andere variabele aanpakken.